sanleandro/sanleandro_ncricdata.yaml

```diff
@@ -0,0 +1,386 @@
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "SanLeandro"
+    name: "NCRICVehicles"
+
+  NCRICVehicleRecords:
+    fqn: "ol.vehicle"
+    entitySetName: "NCRICVehicleRecords"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatCombineTransform>
+          separator: "_"
+          transforms:
+          - !<com.openlattice.shuttle.transforms.ColumnTransform>
+            column: "VehicleID"
+          - !<transforms.ValueTransform>
+            value: "SanLeandro"
+      vehicle.licensenumber:
+        type: "vehicle.licensenumber"
+        column: "Plate"
+      vehicle.licensestate:
+        type: "vehicle.licensestate"
+        column: "State"
+      # ol.licenseplateimage:
+      #   type: "ol.licenseplateimage"
+      #   column: "PlateImage"
+      # ol.vehicleimage:
+      #   type: "ol.vehicleimage"
+      #   column: "OverviewImage"
+      ol.datelogged:
+        type: "ol.datelogged"
+        transforms:
+        - !<transforms.CombineDateTimeTransform>
+          dateColumn: "Date"
+          datePattern: ["MM/dd/yyyy", "M/dd/yyyy", "MM/d/yyyy", "M/d/yyyy" ]
+          timeColumn: "Time"
+          timePattern: ["HH:mm:ss", "HH:mm:ss.S"] # confirmed military time
+          timezone: "America/Los_Angeles"
+      ol.locationcoordinates:
+        type: "ol.locationcoordinates"
+        transforms:
+        - !<transforms.GeographyPointTransform>
+          latTransforms:
+          - !<com.openlattice.shuttle.transforms.ColumnTransform>
+            column: "Latitude"
+          lonTransforms:
+          - !<com.openlattice.shuttle.transforms.ColumnTransform>
+            column: "Longitude"
+      publicsafety.agencyname:
+        type: "publicsafety.agencyname"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "San Leandro Police Department"
+      ol.datasource:
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "SanLeandro"
+    name: "NCRICVehicleRecords"
+
+ # associateOnly will be uncommented after the 1st run of the data on prod,
+ # ...and ol.name property will be commented out
+  NCRICAgencies:
+    fqn: "ol.agency"
+    entitySetName: "NCRICAgencies"
+    propertyDefinitions:
+      ol.id: 
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatCombineTransform>
+          separator: "_"
+          transforms:
+          - !<transforms.ValueTransform>
+            value: "San Leandro Police Department"
+          - !<transforms.ValueTransform>
+            value: "SanLeandro"  
+      ol.name:
+        type: "ol.name"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "San Leandro Police Department"
+      ol.datasource:
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "SanLeandro"
+    # associateOnly: true   
```

Comment:
Why commented out?  This will always be the same no?

Reply:
yes, I just need to run it once before adding it. :) This is a new agency